client/app/models/scheduleitem.coffee

```diff
@@ -198,6 +201,17 @@ module.exports = class ScheduleItem extends Backbone.Model
                (edo.isAfter(start) and edo.isBefore(end)) or \
                (sdo.isBefore(start) and edo.isAfter(end)))
 
+    getLastOccurenceDate: ->
+        if @isRecurrent()
+            options = RRule.parseString @get 'rrule'
+            if options.until?
+                return moment options.until
+            else
+                # arbitrary big value
+                return moment().add 10, 'years'
```

Comment:
> Also, I suppose it's what you did but just to be sure: recurring events are only displayed for ten years, not  afterwards. How will it behave in ten years?

Yes, it's arbitrary, I needed a +infinite value which cast to a moment object. I used +10 years from today (execution day).